### PR TITLE
Dice game project re-added after fixes

### PR DESCRIPTION
**Pull request by BarmenduC**
The 2 notified issues regarding the spec.yaml has been fixed. Do mention if something else needs to be changed.

### CODE REVIEW

**Comment by BarmenduC** [comment]
@Pranavtechie @mehulmpt 
I was instructed to add the file after fixes before the 13th of November, which I did.
Is my submission still valid for the hacktoberfest challenge?

**Comment by Pranavtechie** [comment]
@BarmenduC Please remove the .DS_Store file from the boilerplate repo, You might not able to find this fine in you local-system if you did not enable to show hidden files. 

If you're still unable to see this file, you can attempt to delete this file directly from the command line from the github repo directly itself

**Comment by Pranavtechie** [comment]
@BarmenduC Also there are no `style.css` and `script.js` files in your boilerplate repo, then remove those names form the `files` in the `.cdmrc` file

File: dice-game-project/spec.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    This project focuses mostly on HTML, CSS and layouts with CSS alongside a little bit of Javascript.
+
+    You can use any tools you like to help you complete this project. So if you've got something you'd like to practice, feel free to give it a go.
+
+    Your users should be able to:
+
+    - View the optimal layout for the site depending on their device's screen size
+
+    - Download the starter code and go through the README.md file.
```
Comment (Pranavtechie) [inline dice-game-project/spec.yml:19]: Remove this line, as user's don't have to download

File: dice-game-project/spec.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    The `README.md` file will provide further details about the project. The `style-guide.html` file is where you'll find colors, fonts, etc.
```
Comment (Pranavtechie) [inline dice-game-project/spec.yml:21]: Please remove this line as well, there is no `style-guide.html`  or `Readme.md` in your boilerplate repo

File: dice-game-project/step0/Instructions.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Hello there, welcome to the first step of Dice game project in which we intend to develop a 2 player dice game having 2 seperate dice on the screen for both the player, which on the press of a button generates a random number between 1-6 for both the dice and subsequently changes the image of the dice as per the random number that appears.
```
Comment (Pranavtechie) [inline dice-game-project/step0/Instructions.md:1]: for both the players 

`s` is missing here

File: dice-game-project/step1/Instructions.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+-   `<body>` tag: Tells the browser that the content inside the `<body>` tag is the content to be displayed on the browser page.
+-   `<link>` tag: Tells the browser that we want to link an external file with the HTML document, and process it.
+
+It is advised that you go though the [HTML & CSS Fundamentals course](https://codedamn.com/learn/html-css) for a comprehensive and detailed understanding of these tags.
+
+## What will you do?
+
+In this step you are tasked to create a new `style.css` file and link it with the existing `index.html` document.
+
+1. Add a new `<div> </div>` tag inside the `<body>` element. Give the `<div>` element an id `gallery-container`, this element will act as a container and will have all the gallery images inside this element.
```
Comment (Pranavtechie) [inline dice-game-project/step1/Instructions.md:17]: @BarmenduC You've clearly copied this content from the mutliverse gallery project. Please re-write this or we'll reject your project.

File: dice-game-project/step2/Instructions.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Here we are going to link Javascript through External file
+
+We can also create a separate file to hold the code of JavaScript with the (.js) extension and later incorporate/include it into our HTML document using the src attribute of the <script> tag. It becomes very helpful if we want to use the same code in multiple HTML documents. It also saves us from the task of writing the same code over and over again and makes it easier to maintain web pages.
+
+In this example, we will see how we can include an external JavaScript file in an HTML document.
+
+Let's understand through a simple example.
+
+<html>
```
Comment (Pranavtechie) [inline dice-game-project/step2/Instructions.md:15]: Add code block for this using triple backticks (`)

File: dice-game-project/step2/challenges.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+name: "Linking Javascript"
+stepId: step2
+stepBreakDown: 
+  - text: "Create a 'script.js' file in the same directory as the 'index.html'."
+  - text: "Link the JS file to HTML (If you wish to know how, check instruction number 3)"
```
Comment (Pranavtechie) [inline dice-game-project/step2/challenges.yml:5]: Remove the text in brackets, the instruction number as not visible to users, only the step names are visible

File: dice-game-project/step3/Instructions.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+   b) body {
+      color: blue;
+      }
+  ```
+
+2) Background coloring using CSS
+
+```json
+   for example:
```
Comment (Pranavtechie) [inline dice-game-project/step3/Instructions.md:24]: make sure that the code blocks contain proper comments, do not add any normal lines inside the codeblocks

File: dice-game-project/step3/Instructions.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+Below the h1 tag, type player1 and player2 and make sure to get proper spacing between them(this can also be done with the help of css).
+
+Make sure to link your external CSS file with HTML before proceeding with the styling of the webpage.
+
+After all this has been made sure of, use CSS to provide styling (as per the wishes of the end user) to the pieces of text in this webpage. Styling can be of the following types
+
+1) Font coloring using CSS
+  ```json
+   for example:
```
Comment (Pranavtechie) [inline dice-game-project/step3/Instructions.md:11]: Do not add `a)` in the codeblocks directly, they affect the overall way the codeblock is represented. You can add this text in the codeblock by commenting them in the css file using ` /* */`

File: dice-game-project/step3/Instructions.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+     body {
+  background-color: lightblue;
+}
+```
+
+
+Use the images present in the assets folder in the starting files to display the two dice signifying two players having seperate dice.
+
+If you have reached till here, let's move on to the next step.
```
Comment (Pranavtechie) [inline dice-game-project/step3/Instructions.md:33]: Try rephrasing this line to `You're halfway through the project, great job!` or remove the line altogether

File: dice-game-project/step4/Instructions.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+In this step, we are going to use Javascript and its properties to add some necessary behaviour to our dice game.
+
+We need to generate two random numbers between the numbers 1-6 (both included) to simulate the rolling of a dice. This can be shown with the help of the following example :-
+
+```json
```
Comment (Pranavtechie) [inline dice-game-project/step4/Instructions.md:5]: This is not JSON codeblock, this is `javascript` please use `js` instead of `json`

**Review by Pranavtechie — CHANGES_REQUESTED** [review]

**Comment by BarmenduC** [comment]
@Pranavtechie 
All mentioned changes have been made.

**Comment by BarmenduC** [comment]
@Pranavtechie Is it acceptable now?

**Comment by BarmenduC** [comment]
@Pranavtechie @mehulmpt All previously requested changes have been made. Do provide a final confirmation whether the entry is accepted or not. Thanks in advance

File: dice-game-project/step2/challenges.yml [hunk 11]
```diff
@@ -0,0 +1,5 @@
+name: "Linking Javascript"
+stepId: step2
+stepBreakDown: 
+  - text: "Link the JS file to HTML. If you wish to know how, check instruction number 3"
```
Comment (Pranavtechie) [inline dice-game-project/step2/challenges.yml:4]: **Remove the Instruction Number 3 text and mentioned if the step name**

File: dice-game-project/step3/Instructions.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+      color: blue;
+      }
+  ```
+
+2) Background coloring using CSS
+   for example:
+
+```
+body {
+background-color: lightblue;
```
Comment (Pranavtechie) [inline dice-game-project/step3/Instructions.md:26]: Add appropriate indentation here

File: dice-game-project/step4/Instructions.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+In this step, we are going to use Javascript and its properties to add some necessary behaviour to our dice game.
+
+We need to generate two random numbers between the numbers 1-6 (both included) to simulate the rolling of a dice. This can be shown with the help of the following example :-
+
+```js
+let x = Math.floor((Math.random() *5) + 1);
+```
+
+Math.random() function in Javascript generates a random number between 0 and 1. Multiplying it by 5 we get a random number between 0 and 5 which might be a decimmal number. When combined with math.floor(), we get a random whole number between 0 and 5 and hence on adding 1, we finally get a random integer between 1 to 6.
```
Comment (Pranavtechie) [inline dice-game-project/step4/Instructions.md:9]: Wrap inline code in backticks

**Comment by BarmenduC** [comment]
@Pranavtechie @mehulmpt 
All mentioned changes have been implemented.
Please perform check for merge

**Comment by BarmenduC** [comment]
@Pranavtechie @mehulmpt 
Please review changes.

**Comment by BarmenduC** [comment]
@Pranavtechie @mehulmpt Will this entry not be acceptable anymore?
If yes, please review and merge.

**Pull request closed without merging**